src/common/components/layout/header/Header.module.css

```diff
@@ -1,57 +1,16 @@
 .header {
-  display: inherit;
-  grid-template-columns: inherit;
   grid-column: viewport;
   grid-row: navigation;
-  padding: 10px 0;
-}
-.header > * {
-  grid-column: container;
 }
 
-.content {
-  align-items: center;
+.fakeAction {
```

Comment:
By default HDS does not support buttons in the actions section of the menu. Note to self: test this on mobile after profile deletion works again.

Reply:
This seems to be working fine on mobile